modules/pipeline/services/pipelinesvc/create_task.go

```diff
@@ -53,9 +53,10 @@ func (s *PipelineSvc) makeNormalPipelineTask(p *spec.Pipeline, ps *spec.Pipeline
 	// task.Extra.Envs
 	// task.Extra.Labels
 	// task.Extra.Image
+	task.Extra.Action = *action
 
 	// set executor
-	executorKind, executorName := s.judgeTaskExecutor(action, passedDataWhenCreate.GetActionJobSpecs(extmarketsvc.MakeActionTypeVersion(action)))
+	executorKind, executorName := s.judgeTaskExecutor(task, passedDataWhenCreate.GetActionJobSpecs(extmarketsvc.MakeActionTypeVersion(action)))
```

Comment:
you have to test that running-tasks with old `SCHEDULER` executor.
does these tasks can be continuous reconcile after this pr merged?

Reply:
impl `GetExecutorName` func, this func is compatible with old scheduler executor
![image](https://user-images.githubusercontent.com/30427474/158529747-39be8c98-2f6d-4329-a998-3e5f16d660ae.png)
![image](https://user-images.githubusercontent.com/30427474/158529878-f6adbc2c-9a3d-45fe-8ca1-183c4c92c7e3.png)

Reply:
added ut in pipeline_task.go

Reply:
tested, both old scheduler and new k8sjob is success
![image](https://user-images.githubusercontent.com/30427474/158532961-39e11634-6079-4ef1-a409-b8f1688582f3.png)